src/components/AboutUs/AboutUs.tsx

```diff
@@ -0,0 +1,69 @@
+import { ReactElement } from 'react';
+import { AboutUsIcon } from '../AboutUsIcons/AboutUsIcons';
+
+export const AboutUs = (): ReactElement => {
+  return (
+    <div className="py-15 px-20 ">
```

Comment:
let's change this div to the `section` tag to make this layout for semantic friendly